cmdstanpy/install_cxx_toolchain.py

```diff
@@ -297,7 +300,10 @@ def main():
 
     install_dir = vars(args)['dir']
     if install_dir is None:
-        install_dir = os.path.expanduser(os.path.join('~', '.cmdstanpy'))
+        cmdstan_dir = os.path.expanduser(os.path.join('~', _DOT_CMDSTAN))
```

Comment:
same here

Reply:
took another pass at the logic - what do you think?